docs/docs/storybook-builder/migration-to-storybook-7.md

```diff
@@ -0,0 +1,117 @@
+# Storybook Builder >> Migration to Storybook 7 ||4
+
+This guide explains how to migrate from [`@web/dev-server-storybook` plugin](../dev-server/plugins/storybook.md) (that used opinionated Storybook 6 bundle `@web/storybook-prebuilt`) to Storybook 7 and new Web Storybook Builder.
+
+Most of the [official migration guide for Storybook 7](https://storybook.js.org/docs/web-components/migration-guide) applies to this migration too, with a few additions and exceptions.
+Please check the official guide, but before running the `upgrade` command read the sections below.
+
+## Run upgrade CLI
+
+The Storybook CLI `upgrade` command is not aware of `@web/dev-server-storybook` plugin and as a result it won't work by default.
+But with a simple workaround you can make it work.
+
+First install old version of `builder-vite` and old Storybook 6 renderer for the technology you used in the old setup, e.g. for Web Components it will be:
```

Comment:
```suggestion
First, install the old version of `builder-vite` and old Storybook 6 renderer for the technology you used in the old setup, e.g. for Web Components it will be:
```